subscriptions/src/test/subscriptions.spec.ts

```diff
@@ -0,0 +1,191 @@
+import test from 'ava';
+import * as im from 'immutable';
+import { stub } from 'sinon';
+import { concat, EMPTY, NEVER, of, Subject } from 'rxjs';
+import { delay, take, tap, toArray } from 'rxjs/operators';
+import { Patch, Version } from '@binaris/shift-interfaces/dist/subscriptions';
+import {
+  DEREGISTER,
+  NonEmptyArray,
+  poll,
+  Poller,
+  PollerInputWithObserver,
+  pollReducer,
+  publish,
+  publishInputMapper,
+  REGISTER,
+  serverUpdates,
+} from '../subscriptions';
+
+// tslint:disable-next-line:variable-name
+const Version = (major: number, minor: number): Version => [major, minor];
```

Comment:
If this were a `function` you wouldn't need to `tslint:disable`.

Reply:
```python
True
```
```js
true
```